utils/src/main/scala/za/co/absa/enceladus/utils/validation/ValidationException.scala

```diff
@@ -15,7 +15,8 @@
 
 package za.co.absa.enceladus.utils.validation
 
-class ValidationException(val msg: String, val errors: Seq[String]) extends Exception(msg)
+class ValidationException(val msg: String, val errors: Seq[String])
+  extends Exception(s"$msg Due to errors: ${errors.mkString(",")}")
```

Comment:
**For discussion:** The `ValidationException` is used in `StandardizationInterpreter.validateSchemaAgainstSelfInconsistencies` where logging is done as well. I am not sure, it's a good idea to include the errors in the message here as well. 🤔

Reply:
In cases where the exception is expected, it is easy to query the errors. On the other hand, when it is the underlying case at the top of the stack to cause the process to exit (specifically without direct means to debug), it is much less convenient.

I have summed it up in https://github.com/AbsaOSS/enceladus/pull/1370#issuecomment-638891307

Are there any other ideas on the topic or maybe a middle ground?